### ISSUE TITLE
MD013 `semantic-line-breaks`: Formatter break tables in a list item

### ISSUE THREAD

**Issue by tahv** [rvben/rumdl#590]
### rumdl version

0.1.83

### Installation method

Other (uv)

### Bug description

Formatter break tables in a list item when `semantic-line-breaks` is used.

### Steps to reproduce

1. Create `repro.md` + the `pyproject.toml` below.

```md
- A list item.

    | Lorem ipsum | dolor sit amet             |
    | ----------- | -------------- |
    | consectetur adipiscing elit | sed do eiusmod tempor incididunt ut labore |
```

2. Format the document

### Expected behavior

My expectation is the formatter should "ignore" the line length and not reflow the table cells.

```md
- A list item.

    | Lorem ipsum | dolor sit amet |
    | ----------- | -------------- |
    | consectetur adipiscing elit | sed do eiusmod tempor incididunt ut labore |
```

### Actual behavior

Formatting the document with the configuration below give me this result:

```md
- A list item.

    | Lorem ipsum | dolor sit amet | | ----------- |
    -------------- | | consectetur adipiscing elit | sed do
    eiusmod tempor incididunt ut labore |
```

### Configuration

`pyproject.toml`:

```toml
[tool.rumdl]
line_length = 60
flavor = "standard"
MD013.reflow = true
MD013.reflow-mode = "semantic-line-breaks"
```

### Operating system

Windows

### Additional context

Note that tables that are not list items are not re-formatted (at least not with this configuration, i don't know if there are MD-rules for formatting tables).

**Comment by mattiasgronlund** [comment]
This issue seems to be fixed in version 0.1.85

**Comment by tahv** [comment]
That is correct, fixed by fc3f89d it seem.
Thanks ! 🎉

**Comment by rvben** [comment]
Thanks for the confirmation @tahv ! 🚀
